.pre-commit-config.yaml

```diff
@@ -17,6 +17,7 @@ repos:
         language_version: python3
         args: [--target-version=py38]
         files: ^(python/.*|benchmarks/.*)$
+        exclude: ^python/nx-cugraph/
```

Comment:
Am I correct in assuming this means we won't have `black` formatting for all of nx-cugraph?  Was this intentional?

Reply:
yes intentional, no it doesn't mean we won't have black. It means we won't use black 22.10.0 as in this file. Instead, we use black 24.1.1 as in python/nx-cugraph/lint.yaml. I prefer the linting of the more recent black.

Reply:
An alternative would be to update the versions here in `.pre-commit-config.yaml`